spec/std/string_spec.cr

```diff
@@ -627,6 +627,33 @@ describe "String" do
     end
   end
 
+  describe "rpartition" do
+    describe "by char" do
+      "hello".rpartition('l').should eq ({"hel", "l", "o"})
+      "hello".rpartition('o').should eq ({"", "", "hello"})
+      "hello".rpartition('h').should eq ({"", "", "hello"})
```

Comment:
Why if the char is at the beginning or at the end of the string is the same as if they were not found at all?

```ruby
> "hello".rpartition("o")
=> ["hell", "o", ""]
> "hello".rpartition("h")
=> ["", "h", "ello"]
```